documentation/src/main/asciidoc/migration/index.adoc

```diff
@@ -8,6 +8,9 @@
 :docinfodir: {docinfodir}
 :docinfo: shared,private
 :title-logo-image: image:hibernate_logo_a.png[align=left,pdfwidth=33%]
+:html-meta-description: Hibernate Search, full text search for your entities - Migration Guide
+:html-meta-keywords: hibernate, search, hibernate search, full text, lucene, elasticsearch, opensearch
+:html-meta-canonical-link: https://docs.jboss.org/hibernate/stable/search/migration/html_single/
```

Comment:
Does a version-less canonical link make sense for migration guides?

Reply:
yeah... I wasn't sure about this one... if it was just a user-facing link I'd say https://hibernate.org/search/documentation/migrate/ would be perfect. But since it is supposed to be a link for "similar content pages"... idk ...

Reply:
just point to itself instead?

Reply:
I'd point to `https://docs.jboss.org/hibernate/search/{major.minor}/migration/html_single/`